x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/schemas/patch_rules_schema.test.ts

```diff
@@ -1063,6 +1063,148 @@ describe('patch rules schema', () => {
     });
   });
 
+  test('You cannot send in an array of actions that are missing "group"', () => {
+    expect(
+      patchRulesSchema.validate<
+        Partial<
+          Omit<PatchRuleAlertParamsRest, 'actions'> & {
+            actions: Array<Omit<AlertAction, 'group'>>;
+          }
+        >
+      >({
+        actions: [{ id: 'id', actionTypeId: 'actionTypeId', params: {} }],
+        rule_id: 'rule-1',
+        risk_score: 50,
+        description: 'some description',
+        name: 'some-name',
+        severity: 'junk',
+        type: 'query',
+        references: ['index-1'],
+        query: 'some query',
+        language: 'kuery',
+        max_signals: 1,
+        version: 1,
+      }).error.message
+    ).toEqual(
+      'child "actions" fails because ["actions" at position 0 fails because [child "group" fails because ["group" is required]]]'
+    );
+  });
+
+  test('You cannot send in an array of actions that are missing "id"', () => {
+    expect(
+      patchRulesSchema.validate<
+        Partial<
+          Omit<PatchRuleAlertParamsRest, 'actions'> & {
+            actions: Array<Omit<AlertAction, 'id'>>;
+          }
+        >
+      >({
+        actions: [{ group: 'group', actionTypeId: 'actionTypeId', params: {} }],
+        rule_id: 'rule-1',
+        risk_score: 50,
+        description: 'some description',
+        name: 'some-name',
+        severity: 'junk',
```

Comment:
Same comment for making severity a valid value as above. You will want all valid values except for the one that you are testing which is invalid.